tests/cephfs/cephfs_scale/max_smallfile_snap.py

```diff
@@ -29,7 +29,7 @@ def collect_ceph_details(client, cmd):
 def run(ceph_cluster, **kw):
     """
     Test Cases Covered:
-    CEPH-83573520	Validate the max snapshot that can be created under a root FS sub volume level.
+    CEPH-83574957	Create subvolume and create snapshots until we hit eexception
```

Comment:
Need Snapshot under subvolume group here?

Reply:
in script we have not created any subvolume group.
So not mentioned it

Reply:
subvolume group snapshot feature is yet to come. It's not needed now